apps/frontend/src/components/cards/StatusCardRow.vue

```diff
@@ -112,19 +136,29 @@ export default class StatusCardRow extends Vue {
     ];
   }
 
-  get errorProps(): CardProps | null {
+  get profileErrorProps(): CardProps | null {
     // Want to ignore existing status filter
-    let filter = {
+    const filter = {
       ...this.filter,
       status: undefined
     };
     return {
-      icon: 'alert-circle',
       title: 'Profile Errors',
       subtitle: `Errors running test - check profile run privileges or check with the author of profile`,
       color: 'statusProfileError',
       number: StatusCountModule.countOf(filter, 'Profile Error')
     };
   }
+
+  get waivedProfiles(): CardProps | null {
+    // Want to ignore existing status filter
+    const count = StatusCountModule.countOf(this.filter, 'WaivedTests');
+    return {
+      title: 'Waived Tests',
+      subtitle: `Has ${count} waived tests.`,
```

Comment:
`Consider using an overlay or manual attestation to properly address this control.`